### ISSUE TITLE
Purpose of "answer_0xx" comments

### ISSUE THREAD

**Issue by mathiasbruun** [abjer/sds2019#5]
Just a quick question on the formatting of the Assignment 0 notebook:

What is the purpose of the comments at the top of the exercise cells? Are we supposed to actually fill them in (i.e. `# answer_0xx = whatever the value of the variable is`) as well as just passing the assertions?

**Comment by hyfox** [comment]
Hi Mathias

I think the comments are there to show you, where to paste the data. The locked verification cell underneith the part for student input, then validates the variable (through the `assert` command). Obviously they need to be uncommented to work :)

**Comment by mathiasbruun** [comment]
Ah, that makes sense! :) So basically a template of sorts. I was just thrown off by the `YOUR CODE HERE` because it seemed sort of redundant to include both.

**Comment by Kristianuruplarsen** [comment]
Correct. The #answer_0xx is simply there to remind you what the variable name of your result should be for the assertions to run correctly.

**Comment by GuidoTurdera** [comment]
Hi you all! The same thing goes for the `raise NotImplementedError()`, right? I mean, for the deliverable we should just delete it so our piece of code stays clean?

**Comment by BjornCilleborg** [comment]
> Hi you all! The same thing goes for the `raise NotImplementedError()`, right? I mean, for the deliverable we should just delete it so our piece of code stays clean?

I just commented it out, but I would assume you can delete it.

**Comment by Kristianuruplarsen** [comment]
Either comment it out or delete it when you answer the problems. It is there to help you notice any problems you might have skipped by accident.
